circllhist.js

```diff
@@ -252,7 +252,16 @@ function circllhist() {
     }
     return sum;
   }
-  
+
+  function hist_count(hist) {
+    var count = 0;
+    for(var i=0; i<hist.bvs.length; i++) {
```

Comment:
```suggestion
    for(var i=0, len = hist.bvs.length; i < len; i++) {
```
Avoid property lookup in the loop

Reply:
@bdunavant I just committed the change you suggested.